src/contacts/EditContact/EditContact.js

```diff
@@ -143,16 +148,27 @@ class EditContact extends Component {
                   <Col xs={3}>
                     <Field
                       component={Select}
+                      dataOptions={[{ label: '', value: '' }]}
```

Comment:
It seems for me we have somewhere constant empty item

Reply:
Updated. Thanks